quickwit/quickwit-query/Cargo.toml

```diff
@@ -27,6 +28,7 @@ whichlang = { workspace = true, optional = true }
 
 quickwit-common = { workspace = true }
 quickwit-datetime = { workspace = true }
+lazy_static = "1.5.0"
```

Comment:
for simplicity decided to put the crate here, if we agree with the approach will move it to the parent `Cargo.toml` and reflect the change in `Cargo.lock`